rwengine/src/render/GameRenderer.hpp

```diff
@@ -140,8 +140,9 @@ class GameRenderer {
     void setupRender();
     void renderPostProcess();
 
-    Renderer* getRenderer() {
-        return renderer.get();
+    Renderer*& getRenderer() {
```

Comment:
thats not quite right, should just be `Renderer& getRenderer() {`

And then anything that called it that was doing getRenderer->Something() should be changed to getRenderer.Something().